docs/curators/CURATOR_TOOLS_BASTION.md

```diff
@@ -0,0 +1,249 @@
+Created 102498.md5
+Created 102498.filesizes
+```
+
+Check the contents of the two files:
+```
+[peterl@ip-10-99-0-95 user5]$ more 102498.filesizes 
+5124    ./readme_102498.txt
+301     ./DLPFC_69_72_VNS_results.csv
+332     ./E2_VNS_Ground_Truth.csv
+
+[peterl@ip-10-99-0-95 user5]$ more 102498.md5 
+2b74aa5af1b67e48f0317748cbfdf310  ./readme_102498.txt
+dc1feb8af3b8c02b0b615e968b87786d  ./DLPFC_69_72_VNS_results.csv
+b5a7e0953d1581077c13818153371918  ./E2_VNS_Ground_Truth.csv
+```
+
+## 4. filesMetaToDb
+
+> [!IMPORTANT]
+> To execute the `filesMetaToDb` command, change directory to the dataset's user dropbox directory located at `/share/dropbox/`:
+```
+[peterl@ip-10-99-0-95 ~]$ cd /share/dropbox/user5
+```
+
+The `fileMetaToDb` script can use `102498.filesizes` and `102498.md5` to update file metadata in the database from the user dropbox folder:
+```
+[peterl@ip-10-99-0-95 ~]$ sudo filesMetaToDb 102498
+Updating md5 checksum values as file attributes for 102498
+Number of changes: 3
+Updating file sizes for 102498
+Number of changes: 3
+Updated file metadata for 102498 in database
+```
+
+You should check the adminfile pages of the files associated with this dataset to see if MD5 values and file sizes are visible.
+
+## 5. Go to dataset admin page on gigadb.org
+
+With the post upload operations complete, you need to go back to the page at `https://gigadb.org/adminDataset/update/id/<dataset_id>` in order to continue curation work on the dataset. You will be able to find this link by entering the dataset's DOI, e.g. 102498 into the DOI column header in /adminDataset/admin page.
+
+On the dataset admin page, you will be able to create a mockup page in order to preview the final dataset view page with the information that was added to the database in the previous steps.
+
+## 6. `transfer` - copy dataset files into Wasabi
+
+When all files in a dataset have been finalised and curated then they can be copied into Wasabi using the `transfer` tool. The path to the user dropbox directory is provided as the `--sourcePath` parameter with the value of the `--doi` parameter being the DOI for the dataset. The `--wasabi` flag inform the `transfer` tool to copy files into Wasabi storage. The `--apply` flag takes the `transfer` tool out of dry-run mode that results in the actual transfer of files into Wasabi storage from the user drop box directory.
+```
+[peterl@ip-10-99-0-95 user5]$ transfer --doi 102498 --sourcePath /share/dropbox/user5/ --wasabi --apply
+```
+
+## 7. Housekeeping of user dropboxes of published datasets
+
+After dataset files have been copied into Wasabi, the files should also be backed up into S3 Glacier:
+```
+[peterl@ip-10-99-0-95 user5]$ transfer --doi 102498 --sourcePath /share/dropbox/user5/ --backup --apply
+```
+
+After you have confirmed the files are safely stored in Wasabi and Glacier then the `user` and `user.orig` dropbox directories should be deleted to save storage space after the manuscript has been published.
```

Comment:
it would be useful to have a hint as to how to check the Glacier for the stored files? And maybe a link to the rclone docs about checking Wasabi content)

Reply:
I have mentioned checking `/var/log/gigadb/transfer.log` to see files have been backed up in S3 Glacier.

It would be better to allow curators to use rclone to list files in the S3 Glacier backup bucket. However, I think this will require cleaning up the rclone and aws configurations on bastion server and it would be better fixed and presented in a separate PR because this PR is already quite big and has been open for a while now.